omod/src/main/web/dashboardwidgets/latestobsforconceptlist/latestobsforconceptlist.spec.js

```diff
@@ -27,14 +27,24 @@ describe('LatestObsForConceptList', () => {
     });
 	
 	it('should query for concepts from config', () => {
-		let bindings = {config: { maxAge: '2w', concepts: 'uuid-1, uuid-2', patientUuid: 'patientUuid' }};
+		let bindings = {config: { maxAge: '2w', concepts: 'uuid-1,uuid-2', patientUuid: 'patientUuid' }};
 		let ctrl = $componentController('latestobsforconceptlist', {$scope}, bindings);
```

Comment:
Aren't we supposed to be backwards compatible?

Reply:
@dkayiwa This comment is not clear - backwards compatible in what way?